e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -1009,7 +1009,7 @@ describeEE("scenarios > embedding > dashboard appearance", () => {
     });
   });
 
-  it("should allow to set locale from the `locale` query parameter", () => {
+  it("should allow to set locale from the `#locale` hash parameter (metabase#50182)", () => {
```

Comment:
All tests that were tested with search param `?locale` need to be migrated to test with hash param and only in EE, since this feature only works with Pro plan.